angularjs/Kanban/Overview.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Kanban - Overview	
-description: Kanban Introduction
+title: Overview	| AngularJS | Kanban | Syncfusion
+description: Learn here about Kanban Introduction in Syncfusion Essential AngularJS Kanban Control, its elements, and more.
```

Comment:
Change like this - Learn here all about overview of Syncfusion Essential AngularJS Kanban Control, its elements, and more.

Reply:
Done